libs/designsystem/src/lib/components/page/page.component.scss

```diff
@@ -16,7 +22,7 @@ ion-toolbar {
 
   @include utils.media('>=large') {
     @include utils.not-touch {
-      max-width: 768px;
+      // max-width: 768px;
```

Comment:
Can we remove this whole block?

Reply:
Nooo, missed that one in my self-review 😄 Should be gone from the surface of the earth by now 😃